backend/src/main/java/com/benine/backend/http/RecallPresetHandler.java

```diff
@@ -57,13 +56,17 @@ public void handle(HttpExchange exchange) throws IOException {
    * @param preset the preset used with the values for moving the camera.
    * @throws CameraConnectionException exception thrown when camera cannot connect.
    */
-  public void movingCamera(IPCamera ipcamera, Preset preset) throws CameraConnectionException {
-    Position position = preset.getPosition();
-    ipcamera.moveTo(position, preset.getPanspeed(), preset.getTiltspeed());
-    ipcamera.zoomTo(preset.getZoom());
-    ipcamera.moveFocus(preset.getFocus());
-    ipcamera.setAutoFocusOn(preset.isAutofocus());
-    ipcamera.setIrisPos(preset.getIris());
-    ipcamera.setAutoIrisOn(preset.isAutoiris());
+  public void movingCamera(IPCamera ipcamera, Preset preset, HttpExchange exchange) {
+    try {
```

Comment:
Yes, I think that is a better solution.
Since you already have a try catch in the handle method you can even do it like: 
try { 
....
  movingCamera(ipcamera, preset)
} catch( SQLException | CameraConnectionException e ) { 
  ...
}